memorious/operations/documentcloud.py

```diff
@@ -32,24 +34,25 @@
 
 
 def documentcloud_query(context, data):
-    host = context.get("host", DEFAULT_HOST)
+    host = context.get("host", API_HOST)
     instance = context.get("instance", DEFAULT_INSTANCE)
     query = context.get("query")
     page = data.get("page", 1)
 
-    search_url = urljoin(host, "search/documents.json")
+    search_url = urljoin(host, "/api/documents/search")
     res = context.http.get(
         search_url, params={"q": query, "per_page": 100, "page": page}
     )
-    documents = res.json.get("documents", [])
+
+    documents = res.json.get("results", [])
     for document in documents:
         doc = {
             "foreign_id": "%s:%s" % (instance, document.get("id")),
-            "url": document.get("pdf_url"),
-            "source_url": document.get("canonical_url"),
+            "url": "{}/documents/{}-{}".format(DOCUMENT_HOST, document.get("id"), document.get("slug")),
+            "source_url": "{}/documents/{}/{}.pdf".format(ASSET_HOST, document.get("id"), document.get("slug")),
             "title": document.get("title"),
-            "author": document.get("author"),
-            "file_name": os.path.basename(document.get("pdf_url")),
+            # "author": document.get("author"),
```

Comment:
I think using the `organization.name` as `author` would be more appropriate than using the `user` name. We can avoid the extra api call by passing in `expand=organization` in the search params: https://www.documentcloud.org/help/api#expandable-fields